tests/test_stats_manager_extended.py

```
@@ -0,0 +1,425 @@
+            manager.record_job(10.0, "RSS", "", accepted=False)
+
+            assert manager.session.jobs_found == 1
+            assert "" in manager.by_language
+
+    def test_multiple_saves(self):
+        """Test multiple save operations don't corrupt data."""
+        with tempfile.TemporaryDirectory() as tmpdir:
+            path = pathlib.Path(tmpdir) / "stats.json"
+            manager = StatsManager(stats_path=path)
+
+            for i in range(10):
```

Ruff (0.14.14)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)